tensorflow_datasets/image_classification/wake_vision/wake_vision_dataset_builder.py

```diff
@@ -0,0 +1,211 @@
+"""wake_vision dataset."""
+
+import tensorflow_datasets.public_api as tfds
+import os
```

Comment:
Fix imports:
https://www.tensorflow.org/datasets/contribute?_gl=1*9d64n*_up*MQ..*_ga*MTgwMjYxNjY4OS4xNzIzMDM5OTU3*_ga_W0YLR4190T*MTcyMzAzOTk1Ny4xLjAuMTcyMzAzOTk1Ny4wLjAuMA..#check_your_style_guide

Reply:
I assume that you refer to this: https://github.com/google/styleguide/blob/gh-pages/pyguide.md#313-imports-formatting

In that case I reordered the imports to first import standard libraries and afterwards tfds.